service.php

```diff
@@ -66,10 +64,10 @@ function page_optimize_service_request() {
 	}
 
 	$output = page_optimize_build_output();
-	// TODO: It seems insecure to indiscriminately cache and send headers. Cache only specific headers.
```

Comment:
Only caches stuff sent from the page generation below.

Reply:
I think that is true in most cases, but it is technically possible for another plugin to run first and add headers. In WordPress, I'm not comfortable assuming we're the only one and in total control.

Reply:
I'd return an array from `page_optimize_build_output` with `content` and `headers` so we can cache.

If you don't get to it today, will do tomorrow. :bowing_man:

Reply:
Good idea! Now that you've said it, it feels like common sense :)